### PR TITLE
Orchid thunderjet

### PR DESCRIPTION
**Pull request by IhorBohdan**

### CODE REVIEW

File: cypress/e2e/orders/orders.unreceivePiece.cy.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ describe('orders: Unreceive piece from Order', () => {
         Receiving.checkReceivedPiece(0, caption, barcode);
         // Unreceive piece
         Receiving.unreceivePiece();
-        Receiving.checkUnreceivedPiece(1, caption);
+        // Receiving.checkUnreceivedPiece(1, caption);
```
Comment (penfriend) [inline cypress/e2e/orders/orders.unreceivePiece.cy.js:59]: why have you commented this line?

File: cypress/e2e/orders/test-acquisition-unit-for-orders.cy.js [hunk 2]
```diff
@@ -49,10 +50,10 @@ describe('ui-finance: Orders', () => {
   });
 
   after(() => {
+    console.log(orderId);
```
Comment (penfriend) [inline cypress/e2e/orders/test-acquisition-unit-for-orders.cy.js:53]: Do you need console.log?

Comment (IhorBohdan) [reply cypress/e2e/orders/test-acquisition-unit-for-orders.cy.js]: fixed

File: cypress/e2e/orders/orders.unreceivePiece.cy.js (hunk 1 above)
Comment (IhorBohdan) [reply cypress/e2e/orders/orders.unreceivePiece.cy.js]: Fixed

**Review by TetianaParanich — APPROVED** [review]

**Review by penfriend — APPROVED** [review]

**Pull request merged**